model/task_lifecycle.go

```diff
@@ -1198,8 +1201,74 @@ func UpdatePatchStatus(p *patch.Patch, versionStatus string) error {
 		if err = p.UpdateStatus(patchStatus); err != nil {
 			return errors.Wrapf(err, "updating patch '%s' with status '%s'", p.Id.Hex(), patchStatus)
 		}
+		if p.IsGithubPRPatch() {
+			if err = SendVersionStatusToGithub(p.Id.Hex(), p.GithubPatchData.BaseOwner, p.GithubPatchData.BaseRepo, p.GithubPatchData.HeadHash, "patch status change", "pr-task-reset"); err != nil {
+				return errors.Wrapf(err, "sending patch '%s' status to Github", p.Id.Hex())
+			}
+		}
+	}
+
+	return nil
+}
+
+func SendVersionStatusToGithub(id, owner, repo, ref, desc, job string) error {
```

Comment:
This may belong better in this file https://github.com/evergreen-ci/evergreen/blob/580ebc277164a4d5442b14d6c14f20b4bb239fa6/thirdparty/github.go#L910

Reply:
Also this takes in enough inputs that a struct would be better. We also need to comment this function since it's public